docker-compose.yml

```diff
@@ -4,10 +4,35 @@ services:
   web:
     build: .
     command: "yarn start:dev"
+    environment:
+      DATABASE_HOST: db
+      DATABASE_PORT: 3306
+      DATABASE_USER: user
+      DATABASE_PASSWORD: password
+      DATABASE_NAME: ppaa
+      DATABASE_LOGGING: true
     ports:
       - 3000:3000
     volumes:
       - .:/usr/src/app
       - node_modules:/usr/src/app/node_modules
+    depends_on:
+      - db
+    links:
+      - db:db
+  db:
+    image: mysql:8.3
+    restart: always
+    environment:
+      MYSQL_DATABASE: 'ppaa'
+      MYSQL_USER: 'user'
+      MYSQL_PASSWORD: 'password'
```

Comment:
No deberíamos levantar todas estas variables desde el .env? Podemos especificar que .env al momento de tirar el docker-compose up

Reply:
Lo mismo con las de arriba

Reply:
Si, totalmente, deje un comentario al respecto

> Docker Compose automatically reads the .env file, but just in case I'm adding them here. I feel it's easier (mostly for the teachers) to not have any manual setup to start the app

La idea es que no haya manual steps, para evitar agregarlos al README nomas 🤷